dojo/tools/dependency_check/parser.py

```diff
@@ -261,11 +209,11 @@ def get_finding_from_vulnerability(self, dependency, related_dependency, vulnera
 
         else:
             mitigation = 'Update {}:{} to at least the version recommended in the description'.format(component_name, component_version)
-            description += '\nFilepath: ' + str(dependency_filepath)
+            description += '\n**Filepath**: ' + str(dependency_filepath)
```

Comment:
```suggestion
            description += '\n**Filepath:** ' + str(dependency_filepath)
```

I know I am picky, just looks a bit nicer.

Reply:
no you're right